GsaGH/Components/0_Model/OpenModel.cs

```diff
@@ -100,7 +100,11 @@ public class OpenModel : GH_OasysDropDownComponent {
         fileName += ".gwb";
       }
 
-      model.Open(fileName);
+      try {
+        model.Open(fileName);
+      } catch (GsaApiException) {
```

Comment:
Why not just have Grasshopper catch this exception?

Reply:
Because i thought the error message is not very informative?!